checkout.php

```diff
@@ -0,0 +1,3 @@
+<?php
+echo "checkout feature";
```

Comment:
please create a class and implement all checkout related features there

Reply:
Resolve the your suggestions